stats/aggregation_value.go

```diff
@@ -188,69 +224,21 @@ func (a *AggregationDistributionValue) incrementBucketCount(f float64) {
 }
 
 func (a *AggregationDistributionValue) multiplyByFraction(fraction float64) AggregationValue {
-	ret := newAggregationDistributionValue(a.bounds)
-	ret.count = int64(float64(a.count)*fraction + 0.5) // adding 0.5 because go runtime will take floor instead of rounding
-	if ret.count == 0 {
-		return ret
-	}
+	// for simplicity, we include the oldest partial bucket when the aggregation
+	// is a distribution. We do not try to have a partial fraction.
 
-	if ret.count == 1 {
-		ret.min = (a.min + a.max) / 2
-		ret.max = ret.min
-		ret.sum = ret.min
-		ret.incrementBucketCount(ret.min)
-		return ret
+	ret := newAggregationDistributionValue(a.bounds)
```

Comment:
Did you miss anything? Parameter `fraction` is not used in this function.

Reply:
This method is part of the interface signature: "type AggregationValue interface".
We decided that for this specific aggregation we will not use the "fraction" for simplicity